lib/Doctrine/Migrations/Configuration/Connection/Loader/ConnectionHelperLoader.php

```diff
@@ -14,30 +14,29 @@
  *
  * @internal
  */
-class ConnectionHelperLoader implements ConnectionLoaderInterface
+final class ConnectionHelperLoader implements ConnectionLoaderInterface
 {
     /** @var string */
     private $helperName;
 
     /** @var  HelperSet */
     private $helperSet;
 
-    public function __construct(?HelperSet $helperSet = null, string $helperName)
+    /** @var ConnectionLoaderInterface */
+    private $fallback;
+
+    public function __construct(?HelperSet $helperSet = null, string $helperName, ConnectionLoaderInterface $fallback)
     {
+        $this->helperSet  = $helperSet ?:  new HelperSet();
```

Comment:
duplicate space after the ternary (weird that this was not catched by CS)

Reply:
True, I expected phpcs to auto fix it